quickwit-search/src/search_response_rest.rs

```diff
@@ -26,7 +26,7 @@ use crate::error::SearchError;
 /// SearchResponseRest represents the response returned by the REST search API
 /// and is meant to be serialized into JSON.
 #[derive(Serialize)]
-#[serde(rename_all = "camelCase")]
+#[serde(rename_all = "snake_case")]
```

Comment:
I think `#[serde(rename_all = "snake_case")]` is unnecessary as all rust variables are `snake_case`

Reply:
Can you remove the `rename_all = "snake_case"` whenever it's unnecessary?

Reply:
Sure.